pkg/scheduler/schedule_one_test.go

```diff
@@ -2143,6 +2149,7 @@ func TestSchedulerSchedulePod(t *testing.T) {
 						"3": framework.NewStatus(framework.Unschedulable, "injecting failure for pod test-filter").WithPlugin("FakeFilter"),
 					},
 					UnschedulablePlugins: sets.New("FakeFilter", framework.ExtenderName),
+					EvaluatedNodes:       3,
```

Comment:
Because I added a new field to `type Diagnosis struct`. 
And in these test cases, we use `reflect.deepequal` to compare two fitError, so it will compare the new field `EvaluatedNodes` too.
It is set to 0 default, which is inconsistent with the real situation.